api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -70,6 +71,15 @@ Workspace toApiWorkspace(
   WorkspaceResponse toApiWorkspaceResponse(
       Workspace workspace, RawlsWorkspaceAccessLevel accessLevel);
 
+  default WorkspaceResponse toApiWorkspaceResponse(
+      DbWorkspace dbWorkspace,
+      RawlsWorkspaceListResponse fcResponse,
+      InitialCreditsExpirationService expirationService) {
+    return toApiWorkspaceResponse(
+        toApiWorkspace(dbWorkspace, fcResponse.getWorkspace(), expirationService),
+        fcResponse.getAccessLevel());
+  }
+
   default List<WorkspaceResponse> toApiWorkspaceResponseList(
```

Comment:
split into two parts.  The lower part is now called by getWorkspaceResponseByFeaturedCategory()